src/actions/query.js

```diff
@@ -148,7 +149,7 @@ export const watchEvent = (firebase, dispatch, { type, path, populates, queryPar
           })
           dispatch({
```

Comment:
This got moved to the dispatch after the populate children dispatches (keeps `isLoaded` from being `true` before all children are set to redux)